### PR TITLE
[enterprise-4.16] OSDOCS-10574: add RN entry

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.16

Issue: https://issues.redhat.com/browse/OSDOCS-10574

Link to docs preview: https://76620--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-notable-technical-changes scroll to relevant entry

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -263,6 +263,12 @@ With this release, the etcd tuning parameters can be set to values that optimize
 * `Standard`
 * `Slower`
 
+[discrete]
+[id="ocp-4-16-remove-dasd-artifact"]
+=== {op-system} `dasd` image artifacts no longer supported for LinuxONE and IBM Z systems
```
Comment (SNiemann15) [inline release_notes/ocp-4-16-release-notes.adoc:268]:
```suggestion
=== {op-system} `dasd` image artifacts no longer supported on {ibm-z-name} and {ibm-linuxone-name} (`s390x`)
```

File: release_notes/ocp-4-16-release-notes.adoc [hunk 2]
```diff
@@ -263,6 +263,12 @@ With this release, the etcd tuning parameters can be set to values that optimize
...
+
+With this release, `dasd` image artifacts for `s390x` architectures are removed from the {product-title} image building pipeline. You can still use the `metal4k` image artifact, which contains the same functionality.
```
Comment (SNiemann15) [inline release_notes/ocp-4-16-release-notes.adoc:270]:
```suggestion
With this release, `dasd` image artifacts for the `s390x` architecture are removed from the {product-title} image building pipeline. You can still use the `metal4k` image artifact, which contains the same functionality.
```

**Review by SNiemann15 — CHANGES_REQUESTED** [review]

**Comment by jldohmann** [comment]
@mike-nguyen can you PTAL at this docs PR for QE ack? it's a RN entry about stopping production of `dasd` image artifacts. thank you!

**Pull request merged**